helmclient.go

```diff
@@ -541,6 +541,24 @@ func (c *Client) ListReleaseContents(ctx context.Context) ([]*ReleaseContent, er
 	return contents, nil
 }
 
+// LoadChart loads a Helm Chart and returns relevant parts of its structure.
+func (c *Client) LoadChart(ctx context.Context, chartPath string) (*Chart, error) {
+	helmChart, err := chartutil.Load(chartPath)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
```

Comment:
Nit: I'd put the stuff below in the `newChart` constructor.

Reply:
I don't follow here. You mean add a newChart function?

Reply:
Something like this https://github.com/giantswarm/devctl/blob/be38aacbff4b0409211ad2fd36e5803b0f99cc58/pkg/githubclient/types.go#L18